packages/core/admin/admin/src/components/NpsSurvey/index.js

```diff
@@ -1,14 +1,52 @@
-import * as React from 'react';
-
-import { Flex, Button, Typography } from '@strapi/design-system';
-import { usePersistentState } from '@strapi/helper-plugin';
+import React, { useState } from 'react';
+
+import { Box, Flex, IconButton, Button, Typography, Textarea, Portal } from '@strapi/design-system';
+import { Cross } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { useNpsSurveySettings } from './hooks/useNpsSurveySettings';
+
+const BannerWrapper = styled(Flex)`
+  border: 1px solid ${({ theme }) => theme.colors.primary200};
+  background: ${({ theme }) => theme.colors.neutral0};
+  box-shadow: ${({ theme }) => theme.shadows.filterShadow};
+  padding: ${({ theme }) => theme.spaces[4]};
+  flex-direction: column;
+  position: fixed;
+  bottom: 0;
+  left: 50%;
+  -webkit-transform: translateX(-50%);
+  transform: translateX(-50%);
+  z-index: 9;
+`;
+
+const Header = styled(Box)`
+  margin: 0 auto;
+`;
+
+const RatingButtonWrapper = styled(Button)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: ${32 / 16}rem;
+  width: ${32 / 16}rem;
+  padding: ${({ theme }) => theme.spaces[2]};
+
+  &.selected {
+    background-color: ${({ theme }) => theme.colors.neutral0};
+    border-color: ${({ theme }) => theme.colors.primary700};
+  }
+`;
```

Comment:
`Button` already extends `Flex` at some point, so you can probably apply a lot of these styles as props as well :)